src/canopy_calcs.F90

```diff
@@ -1273,15 +1272,14 @@ SUBROUTINE canopy_calcs(nn)
                         end if
 ! ... user option to calculate in-canopy biogenic emissions
                         if (ifcanbio) then
-                            if (cszref .ge. 0.0_rk .and. dswrfref .gt. 0.0_rk &
```

Comment:
Compiling/running in DEBUG mode leads to an error:  

Program received signal SIGFPE: Floating-point exception - erroneous arithmetic operation.

Backtrace for this error:
...
/src/canopy_bioemi_mod.F90:226

Likely this is the case with not having the daytime only limits on calculating biogenic emissions, which is good now especially considering new LDF/LIF impacts and emissions at night, but we will need to have conditions inside of the biogenic emissions module to deal with this.